script.js

```diff
@@ -1,38 +1,59 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+class AwesomeBooks {
+  allbooks = [];
+
+  constructor(storedBooks) {
+    this.allbooks = storedBooks;
+  }
+
+  addBook(title, author) {
+    const book = new Book(title, author);
+    this.allbooks.push(book);
+    this.#updateBookStorage();
+  }
+
+  removeBook(index) {
+    this.allbooks.splice(index, 1);
+    this.#updateBookStorage();
+  }
+
+  #updateBookStorage() {
+    localStorage.setItem('data', JSON.stringify(this.allbooks));
+  }
+}
 
 const booklist = document.getElementById('books');
 let BookArray = [];
+let awesomeBooks;
 
 function printScreen() {
-  BookArray = JSON.parse(localStorage.getItem('data'));
+  booklist.innerHTML = '';
+  const bookArray = JSON.parse(localStorage.getItem('data'));
   if (BookArray == null) {
     BookArray = [];
   }
-  booklist.innerHTML = '';
-  BookArray.forEach((value, index) => {
-    const book = `<li> <p> ${value[0]}</p> <p>${value[1]}</p><button onclick="removeBookFroList(${index})">Remove</button><hr></li>`;
-    const liTag = document.createElement('li');
-    liTag.setAttribute('id', index);
-    liTag.innerHTML = book.trim();
-    booklist.appendChild(liTag);
-  });
-}
-
-function updateBookStorage() {
-  localStorage.setItem('data', JSON.stringify(BookArray));
+  awesomeBooks = new AwesomeBooks(bookArray);
+  if (awesomeBooks.allbooks.length > 0) {
+    awesomeBooks.allbooks.forEach((book, index) => {
+      const bookone = `<li> <p> ${book.title}</p> <p>${book.author}</p><button onclick='removeBookFroList(${index})'>Remove</button><hr></li>`;
+      const liTag = document.createElement('li');
+      liTag.setAttribute('id', index);
+      liTag.innerHTML = bookone.trim();
+      booklist.appendChild(liTag);
+    });
+  }
 }
 
 function addBookToList() {
   const titleinput = document.getElementById('title').value;
   const authorinput = document.getElementById('author').value;
-  BookArray.push([titleinput, authorinput]);
-  updateBookStorage();
+  awesomeBooks.addBook(titleinput, authorinput);
   printScreen();
 }
 
 function removeBookFroList(index) {
-  BookArray.splice(index, 1);
-  updateBookStorage();
+  awesomeBooks.removeBook(index);
   printScreen();
 }
 
```

Comment:
- [ ] Add and remove doesn't work, please make sure that adding and removing button works perfectly after refactor with classes.

![storage](https://user-images.githubusercontent.com/55635977/133310901-a4d724fc-1fc4-4870-b56c-8f9432fa87ba.png)

- [ ] Make sure that the localStorage keeps our title and author data.
- [ ] Kindly, fix eslint errors without disable the line of code.
- [ ] Use unique id property for each book since it would make it easier to remove books with duplicate values.

Reply:
This issue is well resilved! ✔️